tests/cpp_api/audio_tests/audio_tests.cpp

```diff
@@ -61,7 +61,7 @@ bool verify_non_silent_region_output(int *nsr_begin, int *nsr_length, std::strin
 
     if (fin.fail()) {
         std::cout << "Error: Failed to read from the input binary file\n";
-        return 1;
+        return 0;
```

Comment:
@fiona-gladwin should the failure return -1?

Reply:
Added changes to return bool variable instead of int

Reply:
Modified the verify output functions in audio unit tests to return int instead of bool, and added changes to return failure as -1